electron/operations/handleError.js

```diff
@@ -5,13 +5,11 @@
 
 const { app, dialog } = require('electron');
 
-const {
-  parity: { channel }
-} = require('../../package.json');
-const parityPath = require('../utils/parityPath');
+const { bugs: { url }, parity: { channel } } = require('../../package.json');
+const pino = require('../utils/pino')({ name: 'electron' });
```

Comment:
We seem to do ` require('../utils/pino')({ name: 'electron' });` everywhere. Shouldn't `../utils/pino` configure with `{ name: 'electron' }` and just return that?

Reply:
Except [here](https://github.com/parity-js/light/blob/am-robust-electron/electron/operations/runParity.js#L16).

I could have done `const { pino, parityPino } = require('../utils/pino')`, but it seemed to me that this method is slightly more flexible: each JS module could have its own name/logLevel/other options. It also follows their usage: https://github.com/pinojs/pino#usage

I just pushed a commit where `{ name: 'electron' }` is the default parameter, so that we use it like `require('../utils/pino')()`